search/filters.py

```diff
@@ -0,0 +1,21 @@
+from django_filters import filters
+from django_filters.rest_framework import FilterSet
+
+
+class CompanyFilter(FilterSet):
+    name = filters.CharFilter(method="by_name_filter")
+    region = filters.CharFilter(method="by_region_filter")
+
+    def by_name_filter(self, queryset, name, value):
+        if value:
+            name = self.request.query_params.get("name", "").strip()
+            return queryset.filter(name__icontains=name).order_by("name")
```

Comment:
For ordering there are ordering filters.

These filters only have to apply filtering.

Reply:
Thank you for feedback. Deleted ordering.

Reply:
In general, in DRF projects, I strongly recommend to be very careful with usage of `.order_by()`. Or you can end up with cases when you need to have ordering filters, and they don't work because ordering is already applied somewhere else.

Reply:
I've tried. Is this smth that you've meant?